### PR TITLE
Remove unnecessary Glazier dependencies.

### PR DESCRIPTION
**Pull request by xorgy**

### CODE REVIEW

File: src/id.rs [hunk 1]
```diff
@@ -21,9 +21,12 @@ pub struct Id(NonZeroU64);
 impl Id {
     /// Allocate a new, unique `Id`.
     pub fn next() -> Id {
```
Comment (DJMcNab) [inline src/id.rs:23]: `Id` is kind of an awful name for this type, although this PR isn't really the place to fix it

**Review by DJMcNab — APPROVED** [review]

File: src/id.rs (hunk 1 above)
Comment (xorgy) [reply src/id.rs:23]: Yeah, I think the mechanism is maybe not ideal, and personally think the type should maybe be `Option<NonZeroUsize>` (but that's technically complicated by AccessKit, which wants `u64`), and this patch could introduce a needless runtime check when coercing to NonZero (which is just `unsafe` in Glazier) depending on how aware the compiler is of the static initalizer, but it's fine for now I think.

Comment (nicoburns) [reply src/id.rs:23]: I've turned this into an issue so as to avoid blocking this PR https://github.com/linebender/xilem/issues/168

**Review by nicoburns — APPROVED** [review]
Regarding the `kurbo` imports, I don't love importing from a reexport like this. There are a number of alternative strategies we could use:
- We could re-export `vello::kurbo` as `xilem::kurbo` and then have everything import from `crate::kurbo`? The re-export could be private (`pub (crate)`) if we don't want to publicly re-export.
- We could depend directly on `kurbo` in `xilem`
- If `kurbo` use in `xilem` is mainly for interaction with `vello`, we could re-export individual types from `kurbo` in the top-level `vello` namespace (so that we can do `use vello::Circle`)

But that can be resolved in a future PR. This one solves the immediate issue with glazier imports :)

**Comment by xorgy** [comment]
Yeah, regarding the reexports: we already directly import kurbo, and I will be using it for now in the bits of functionality I'm dragging in from Glazier for the winit port. I think it's not crazy to use the xilem public reexport for now, since xilem is tightly coupled to vello anyhow, but this all could be more elegant some day I'm sure.

**Pull request merged**